languages/ga.ini

```
@@ -468,6 +481,7 @@ export_fail = "Níor easpórtáladh do chuid míreanna"
 export_invalid_format = "Ní féidir an fhormáid easpórtála atá roghnaithe a úsáid i gcás an taifid seo."
 export_missing = "Bhí roinnt sonraí in easnamh. Níor easpórtáladh do chuid míreanna."
 export_no_formats = "Ní féidir an taifead seo a easpórtáil."
+export_records_complete = "{comhaireamh, iolra, =1 {Taifead} eile {#taifid}} easpórtáilte."
```

Looks like part of the markup got translated here:

```suggestion
export_records_complete = "{count, plural, =1 {Taifead} other {# taifid}} easpórtáilte."
```